man/memkind.3

```diff
@@ -313,14 +316,44 @@ is supported by TBB heap manager described in
 .B ENVIRONMENT
 section since Intel TBB 2019 Update 4.
 .PP
+.BR memkind_defrag_reallocate ()
+reallocates the object inside specific
+.IR "kind" ,
+doing so could improve overall external fragmentation of the heap. In case of failure function return
```

Comment:
IF

This reallocates the object conditionally. The condition is that it's worth doing :P

Reply:
Done at least first part. Regarding second part I tried to rephrase it but as always I appreciate a feedback.